packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -274,7 +266,8 @@ export default declare<State>((api) => {
         visitClassNamesPath(path, { context: 'root', state, parentPath: path });
       },
       JSXOpeningElement(path, state) {
-        if (state.usesXcss) {
+        const compiledXCSSProp = state.opts.processXcss ?? true;
+        if (compiledXCSSProp) {
```

Comment:
usesXcss / processXcss are now considered two flows.

- process turns on the compilation
- uses makes compiled bring in the runtime

Reply:
/nit should we rename `useXcss` to `injectXcssRuntime` to be more explicit?